cmd/seed-controller-manager/options.go

```diff
@@ -88,6 +89,9 @@ type controllerRunOptions struct {
 	admissionWebhook                                 webhook.Options
 	concurrentClusterUpdate                          int
 	addonEnforceInterval                             int
+	controlPlaneLeaderElectLeaseDurationSeconds      int
+	controlPlaneLeaderElectRenewDeadlineSeconds      int
+	controlPlaneLeaderElectRetryPeriodSeconds        int
```

Comment:
Nit: why we don't use `metav1.Time` as a type here? That way, the flags would have the same semantics as the flags used by kube-controller-manager and kube-scheduler.

Reply:
Well, `metav1.Time` is to represent a point in time, while here we are representing time durations (in sec). If you take a look at k8s object this is the way it is done.

Reply:
Sorry, I've been talking about `metav1.Duration`, such as here: https://github.com/kubernetes/kubernetes/blob/66cbf0196bd5ed0f06d7c40fccba887aae5405d8/staging/src/k8s.io/component-base/config/types.go#L46-L61

Reply:
ah ok, yes in this case it makes sense

Reply:
I thought a bit about this, and I'm not sure it's necessary to use `metav1.Duration` here, it would be convenient if we had to represent a parameter that can assume very different time scales, but here we will be always dealing with seconds. If you don't have a strong point against I would rather keep this as it is.